base/utils/R/read.output.R

```diff
@@ -102,6 +102,22 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
   if(dataframe==FALSE){
   return(result)
   }else if (dataframe==TRUE){
+    
+    # Check if there are variables that have multiple dimensions 
+    # for example soil moisture at multiple levels.
+    # Currently we don't have a consensus how to convert these to dataframe format
+    # so they should be omitted. 
+    
+    for(var in names(result)){
+      c <- dim(result[[var]])[2]
+      r <- dim(result[[var]])[1]
+      if(!is.na(c) & r > 1){
+        PEcAn.logger::logger.warn("Variable", var, "has", r, "dimensions,
+      it cannot be loaded and will be omitted.")
+        result[[var]] <- NULL 
+      }
+    }
```

Comment:
I think Mike's suggestion was that for variables that have multiple levels (e.g. Soil Moisture), to sum across the layers as a first pass (`colSums`).

Reply:
@mdietze should I include that?

Reply:
I'd say that it probably does make sense for this to be consistent with the earlier PR where I recommended that switch. That said, the end goal should be the ability to safely return matrices and/or a specifically selected layer without downstream code dying